code/modules/surgery/tools.dm

```diff
@@ -261,3 +261,135 @@
 		return TRUE
 	return
+
+/obj/item/scalpel/advanced
+	name = "laser scalpel"
+	desc = "An advanced scalpel which uses laser technology to cut. It's set to scalpel mode."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "scalpel_a"
+	hitsound = 'sound/weapons/blade1.ogg'
+	force = 16
+	toolspeed = 0.25
+	light_color = LIGHT_COLOR_GREEN
+
+/obj/item/scalpel/advanced/Initialize()
+	set_light(1)
+	START_PROCESSING(SSobj, src)
+
+/obj/item/scalpel/advanced/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	return ..()
+
+/obj/item/scalpel/advanced/attack_self(mob/user)
+	playsound(get_turf(user),'sound/machines/click.ogg',50,1)
+	var/obj/item/circular_saw/advanced/saw = new /obj/item/circular_saw/advanced(drop_location())
+	to_chat(user, "<span class='notice'>You incease the power, now it can cut bones.</span>")
+	qdel(src)
+	user.put_in_active_hand(saw)
+
+/obj/item/circular_saw/advanced
+	name = "laser scalpel"
+	desc = "An advanced scalpel which uses laser technology to cut. It's to set saw mode."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "saw_a"
+	hitsound = 'sound/weapons/blade1.ogg'
+	force = 19
+	toolspeed = 0.25
+	sharpness = IS_SHARP_ACCURATE
+	light_color = LIGHT_COLOR_GREEN
+
+/obj/item/circular_saw/advanced/Initialize()
+	set_light(2)
+	START_PROCESSING(SSobj, src)
+	..()
+
+/obj/item/circular_saw/advanced/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	return ..()
+
+/obj/item/circular_saw/advanced/attack_self(mob/user)
+	playsound(get_turf(user),'sound/machines/click.ogg',50,1)
+	var/obj/item/scalpel/advanced/scalpel = new /obj/item/scalpel/advanced(drop_location())
+	to_chat(user, "<span class='notice'>You lower the power.</span>")
+	qdel(src)
+	user.put_in_active_hand(scalpel)
+
+/obj/item/retractor/advanced
+	name = "mechanical pinches"
+	desc = "An agglomerate of rods and gears. It resembles a retractor."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "retractor_a"
+	toolspeed = 0.25
+
+/obj/item/retractor/advanced/attack_self(mob/user)
+	playsound(get_turf(user),'sound/items/change_drill.ogg',50,1)
+	var/obj/item/hemostat/advanced/hemostat = new /obj/item/hemostat/advanced(drop_location())
+	to_chat(user, "<span class='notice'>You set the [src] to hemostat mode.</span>")
+	qdel(src)
+	user.put_in_active_hand(hemostat)
+
+/obj/item/hemostat/advanced
+	name = "mechanical pinches"
+	desc = "An agglomerate of rods and gears. It resembles an hemostat."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "hemostat_a"
+	toolspeed = 0.25
+
+/obj/item/hemostat/advanced/attack_self(mob/user)
+	playsound(get_turf(user),'sound/items/change_drill.ogg',50,1)
+	var/obj/item/retractor/advanced/retractor = new /obj/item/retractor/advanced(drop_location())
+	to_chat(user, "<span class='notice'>You set the [src] to retractor mode.</span>")
+	qdel(src)
+	user.put_in_active_hand(retractor)
+
+/obj/item/surgicaldrill/advanced
+	name = "searing tool"
+	desc = "It projects an high power laser used to mend tissue togheter."

```

Comment:
typo:
togheter -> together
And the description should probably show that it can also be used as a surgical drill. 

And "It's set to to mending mode" plus "It's set to drill mode" like with laser scalpel.